spec/dashboard_spec.js

```diff
@@ -44,7 +44,7 @@ describe('dashboard', () => {
         monitoring.nextSearches();
         monitoring.nextReorder();
         monitoring.saveSettings();
-        browser.sleep(300);
+        browser.sleep(3000);
```

Comment:
it's better to use `wait` so it can run faster if possible, check `spec/helpers/utils.js`

Reply:
yep it's better for sure, done